spec/change_set_persisters/change_set_persister_spec.rb

```diff
@@ -431,6 +431,30 @@
       end
     end
 
+    it "runs characterization for subsequent files", run_real_derivatives: true do
```

Comment:
Can you expand on "subsequent files" here? Maybe "runs characterization for all files when added in sequence with the same persister"? Or something that points just a little bit more to the source of the scenario?